src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
```

Comment:
Consider defining a constant for the length of the `Color` enum values array to avoid calling `Color.values()` multiple times, which creates a new array each time it's called.